Servers/WindowServer/WSWindow.cpp

```diff
@@ -98,18 +99,48 @@ void WSWindow::handle_mouse_event(const WSMouseEvent& event)
     }
 }
 
+void WSWindow::update_menu_item_text(int index)
```

Comment:
It seems a little strange that an index of `0` is treated as a magic number for minimize/restore, and `1` is a magic number for restore/maximize. Maybe we could use an enum (or something else) to make this relationship a bit more obvious?